File: problem001/sol.java
public class sol {
    public static void main(String[] args) {
        int i = 1, sum = 0;
        for (; i < 1000; i++)
            if (i % 3 == 0 || i % 5 == 0) sum += i;
        System.out.println(sum);
    }
}

File: problem001/sol.c
#include <stdio.h>

int main(int argc, char ** argv) {
    int i = 1, sum = 0;

    for (; i < 1000; i++)
        if (!(i % 3 && i % 5)) sum += i;

    printf("%d\n", sum);
    return 0;
}
